### PR TITLE
Button and image

### PR DESCRIPTION
**Pull request by iamalhera**
button

### CODE REVIEW

File: src/components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
+@import url('/src/components/utils.css');
+
+.container-row{
+    justify-content: space-evenly;
```
Comment (FarhanMobashir) [inline src/components/button/button.css:4]: It should also have flex direction row as per your name

**Review by FarhanMobashir — COMMENTED** [review]
All good ✅

File: src/components/button/button.css (hunk 1 above)
Comment (iamalhera) [reply src/components/button/button.css:4]: its default so did not mentioned that, anyway i will do that

Comment (FarhanMobashir) [reply src/components/button/button.css:4]: I know that. But many time you will use this to override pre written css and there it will not work

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Floating buttons should have box-shadow. And just a suggestion image are too big , not looking good

File: src/components/button/button.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    justify-content : center ;
+}
+.btn-icon:hover{
+    font-size : 1.8rem ;
+    border : 1.5px solid var(--danger-text);
+    background-color : var(--white-color);
+    color : var(--danger-text);
+}
+.fas.fa-trash{
+    margin-right : 5px;
```
Comment (amantulsyan35) [inline src/components/button/button.css:96]: rem can be used here as it has been used in font-sizes, will make the code more consistent

**Comment by amantulsyan35** [comment]
Images looks good to merge

File: src/components/button/button.css [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    background-color: var(--ca-primary-color-lighter);
+    font-size : 1.8rem ;
+}
+.btn-success{
+    color : var(--white-color);
+    background-color : var(--success-text);
+}
+.btn-success:hover{
+    background-color: var(--online-color);
+    font-size : 1.8rem ;
```
Comment (nikhiljugale007) [inline src/components/button/button.css:30]: One suggestion Instead of increasing font-size you can use transform property.

**Review by nikhiljugale007 — COMMENTED** [review]
Image component is perfect.
In button component one suggestion is you can categories button and  solid button , outlined button , button with icon. In that category you can have primary button, success button, danger button.

File: src/components/button/button.css (hunk 2 above)
Comment (iamalhera) [reply src/components/button/button.css:96]: but px less than 10 units is fine to use i guess

File: src/components/button/button.css (hunk 3 above)
Comment (iamalhera) [reply src/components/button/button.css:30]: will look in to that soon

**Pull request merged**